john.html

```diff
@@ -0,0 +1,13 @@
+<html>
+<head>
+<meta charset="utf-8">
+<title>John</title>
+</head>
+<body>
+<img src="http://jimmyfungus.tumblr.com/post/115968224086"
+<ul>
```

Comment:
Hmm usually if I see a `<ul>` I expect to see some `<li>`s inside.



Oh no! Please re-read step 8 instructions. Programming is a very detail-oriented profession.